subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/impl/DefaultFileSystemSnapshotter.java

```diff
@@ -277,4 +278,19 @@ public String asNonInterned() {
             return string;
         }
     }
+
+    private static class StripedProducerGuard<T> {
```

Comment:
It looks like this commit should have gone to https://github.com/gradle/gradle/pull/9695, right?

Reply:
Removed.